# Publication Monitoring - Year 2025

## 23/07/2025

> Bakulin, A., Levy, N., Ergen, C., Maaskola, J., & Yosef, N. 
  NOLAN: SELF-SUPERVISED FRAMEWORK FOR MAPPING CONTINUOUS TISSUE ORGANIZATION. 
  In ICLR 2025 Workshop on Machine Learning for Genomics Explorations.
  https://openreview.net/forum?id=DzlhvkuFqz

> Adduri, A., Gautam, D., Bevilacqua, B., Imran, A., Shah, R., Naghipourfar, M., ... & Roohani, Y. H. (2025). 
  Predicting cellular responses to perturbation across diverse contexts with STATE. 
  bioRxiv, 2025-06.
  https://www.biorxiv.org/content/10.1101/2025.06.26.661135v2

> Busarello, E., Biancon, G., Cimignolo, I., Lauria, F., Ibnat, Z., Ramirez, C., ... & Tebaldi, T. (2025).
  Cell Marker Accordion: interpretable single-cell and spatial omics annotation in health and disease. 
  Nature Communications, 16(1), 5399.
  https://www.nature.com/articles/s41467-025-60900-4

> Dong, M., Su, D. G., Kluger, H., Fan, R., & Kluger, Y. (2025). 
  SIMVI disentangles intrinsic and spatial-induced cellular states in spatial omics data. 
  Nature Communications, 16(1), 2990.

> Morrill Gavarró, L., Couturier, D. L., & Markowetz, F. (2025). 
  A Dirichlet-multinomial mixed model for determining differential abundance of mutational signatures. 
  BMC bioinformatics, 26(1), 59.


## 25/06/2025

> Huang, A. C., Hsieh, T. H. S., Zhu, J., Michuda, J., Teng, A., Kim, S., ... & Chu, C. (2025). 
  X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models. bioRxiv, 2025-06.

> Scherer, M., Singh, I., Braun, M. M., Szu-Tu, C., Sanchez Sanchez, P., Lindenhofer, D., ... & Velten, L. (2025). 
[Clonal tracing with somatic epimutations reveals dynamics of blood 
ageing](https://www.nature.com/articles/s41586-025-09041-8). Nature.

> Jaganathan, K., Ersaro, N., Novakovsky, G., Wang, Y., James, T., Schwartzentruber, J., ... & Farh, K. K. H. (2025). 
[Predicting expression-altering promoter mutations with deep 
learning](https://www.science.org/doi/10.1126/science.ads7373).
Science, eads7373.

> Zhang, J., Tian, R., Liu, J., Yuan, J., Zhang, S., Chi, Z., ... & Wang, D. (2024). A two-front nutrient supply environment fuels small intestinal physiology through differential regulation of nutrient absorption and host defense. Cell, 187(22), 6251-6271.

> Saraswat, M., Rueda-Gensini, L., Heinzelmann, E., Gracia, T., Memi, F., de Jong, G., ... & Mall, M. (2025). Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics. bioRxiv, 2025-05.

> Qi, J., Luo, Z., Li, C. Y., Wang, J., & Ding, W. (2025). Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks. Nature Computational Science, 1-12.

> 