# Publication Monitoring - Year 2025


## 3/12/2025

> Liu, S., Tang, J., Ma, J., & Liang, S. (2025). TissueNarrator: 
  Generative Modeling of Spatial Transcriptomics with Large Language Models. bioRxiv, 2025-11.

> Dai, M., Torok, T., Sun, D., Shende, V., Wang, G., Lin, Y., ... & Chen, F. (2025). 
  LARIS enables accurate and efficient ligand and receptor interaction analysis in spatial transcriptomics. bioRxiv, 2025-11.

> CellProphet: Dissecting Virtual Cell Differentiation through AI-Powered Dynamic Gene Regulatory Network Inference. bioRxiv 2025

> Ariad, D., Yan, S. M., Victor, A. R., Barnes, F. L., Zouves, C. G., Viotti, M., & McCoy, R. C. (2021). 
  Haplotype-aware inference of human chromosome abnormalities. Proceedings of the National Academy of Sciences, 118(46), e2109307118.

> Guo, J., Sarafinovska, S., Hagenson, R. A., Valentine, M. C., Dougherty, J. D., Mitra, R. D., & Muegge, B. D. (2025). 
  SMURF reconstructs single-cells from Visium HD data to reveal zonation of transcriptional programs in the intestine. bioRxiv, 2025-05.

> multiVIB: A unified probabilistic contrastive learning framework for atlas-scale integration of single-cell multi-omics data. bioRxiv 2025



## 18/11/2025


> Ding, T., Wagner, S. J., Song, A. H., Chen, R. J., Lu, M. Y., Zhang, A., ... & Mahmood, F. (2025). 
  A multimodal whole-slide foundation model for pathology. Nature Medicine, 1-13.

> Gibbs, C. S., Chen, A., Bonneau, R., & Cho, K. (2025). 
  GLM-Prior: a nucleotide transformer model reveals prior knowledge as the driver of GRN inference performance. bioRxiv, 2025-06.

> Sarwar, A., Rue, M., French, L., Cross, H., Choi, S., Chen, X., & Gillis, J. (2025). 
  Cross-expression meta-analysis of mouse brain slices reveals coordinated gene expression across spatially adjacent cells. Genome Biology, 26(1), 1-35.

> Wu, C. Y., Rong, J., Sathe, A., Hess, P. R., Lau, B. T., Grimes, S. M., ... & Zhang, N. R. (2025). 
  Cancer subclone detection based on DNA copy number in single-cell and spatial omic sequencing data. Nature Methods, 1-11.

> Benjamin, A. S., & Zador, A. (2025). TissueFormer: a neural network for 
  labeling tissue from grouped single-cell RNA profiles. bioRxiv.

> Wang, Z., Zeng, Y., Tan, Z., Chen, Y., Huang, X., Zhao, H., ... & Yang, C. (2025). 
  A unified framework for identification of cell-type-specific spatially variable genes in spatial transcriptomic studies. 
  Proceedings of the National Academy of Sciences, 122(46), e2503952122.

> Gorin, G., Chari, T., Carilli, M., Vastola, J. J., & Pachter, L. (2025). 
  Monod: model-based discovery and integration through fitting stochastic transcriptional dynamics to single-cell sequencing data. Nature Methods, 1-15.

> Schaefer, M., Peneder, P., Malzl, D., Lombardo, S. D., Peycheva, M., Burton, J., ... & Bock, C. (2025). 
  Multimodal learning enables chat-based exploration of single-cell data. Nature Biotechnology, 1-11.

> Huizing, G. J., Samaran, J., Capocefalo, D., Audit, A., Peyré, G., & Cantini, L. (2025). 
  STORIES: learning cell fate landscapes from spatial transcriptomics using optimal transport. Nature Methods, 1-10.

> LeRoy, N. J., Zheng, G., Khoroshevskyi, O., Campbell Jr, D. R., Zhang, A., & Sheffield, N. C. (2025). 
  Atacformer: A transformer-based foundation model for analysis and interpretation of ATAC-seq data. bioRxiv, 2025-11.

> Li, Z., Qu, S., Liang, H., Tang, R., Zhang, X., Lu, F., ... & Chen, G. (2025). 
  Integrative deep learning of spatial multi-omics with SWITCH. Nature Computational Science, 1-13.

> Cheng, M., Jansen, J., Reimer, K. C., Grande, V. P., Nagai, J. S., Li, Z., ... & Costa, I. G. (2025). 
  PHLOWER leverages single-cell multimodal data to infer complex, multi-branching cell differentiation trajectories. Nature Methods, 1-9.



## 5/11/2025

> Zhu, Q., Zheng, Y., Sang, Y., Zhan, Y., Zhu, Z., Ding, J., & Zheng, Y. (2024). 
  Suica: Learning super-high dimensional sparse implicit neural representations for spatial transcriptomics. arXiv preprint arXiv:2412.01124.

> Zhang, Y., Liu, H., Wang, H., Li, Z., Li, Y., Chen, J., ... & Li, M. (2025). 
  CellBin: a generalist framework to process spatial omics data to cell level. bioRxiv, 2025-10.

> VariantFormer: A hierarchical transformer integrating DNA sequences with 
  genetic variations and regulatory landscapes for personalized gene expression prediction

> Williams, M. J., Vázquez-García, I., Tam, G., Wu, M., Varice, N., Havasov, E., ... & Shah, S. P. (2025). 
  Tracking clonal evolution during treatment in ovarian cancer using cell-free DNA. Nature, 1-9.

> Beheler-Amass, M., Jackson, C. A., Gresham, D., & Bonneau, R. (2025). 
  Dynamic Gene Regulatory Network Inference with Interpretable, Biophysically-Motivated Neural ODEs. bioRxiv, 2025-09.

> Wang, S., Leng, L., Wang, Q., Gu, Y., Li, J., An, Y., ... & Lin, G. (2024). 
  A single-cell transcriptome atlas of human euploid and aneuploid blastocysts. Nature Genetics, 56(7), 1468-1481.

> Chen, H., Li, D., & Bar-Joseph, Z. (2023). SCS: cell segmentation for 
  high-resolution spatial transcriptomics. Nature methods, 20(8), 1237-1243.

## 16/10/2025

> Shi, Q., Chen, Y., Li, Y., Qin, S., Yang, Y., Gao, Y., ... & Zhang, Z. (2025).
  Cross-tissue multicellular coordination and its rewiring in cancer. Nature, 1-10.

> Tian, R., Xue, Z., Chen, Y., Qi, Y., Zhang, J., Yuan, J., ... & Liu, W. (2025). 
  Integrating cross-sample and cross-modal data for spatial transcriptomics and 
  metabolomics with SpatialMETA. Nature Communications, 16(1), 8855.

> Bracht, S. A., Rong, J., Gier, R. A., DeMarshall, M., Golden, H., 
  Dhakal, D., ... & Shaffer, S. M. (2025). 
  Mitochondrial clone tracing within spatially intact human tissues. bioRxiv, 
  2025-07.

## 17/09/2025

> Liu, T., Zhang, X., Ying, R., & Zhao, H. (2025). Pre-training Genomic Language 
  Model with Variants for Better Modeling Functional Genomics. bioRxiv, 2025-02.

> Gong, Q., Cao, K., Weir, J. A., Wu, Q., Li, D., & Chen, F. (2025). 
  Energy-Regularized Graph Learning for Multiscale Spatial Representation. 
  bioRxiv, 2025-08.

> SpatialZoomer: multi-scale feature analysis of spatial transcriptomics

> Zhang, N., Long, Y., Xia, S., Jia, P., Jin, Y., Chen, S., ... & Li, M. (2025). 
  A histology foundation model for high-resolution spatial omics prediction,
  tumor detection, and, clustering of spatial transcriptomics. bioRxiv, 2025-08.

> Chitra, U., Dan, S., Krienen, F., & Raphael, B. J. (2025). GASTON-Mix: a 
  unified model of spatial gradients and domains using spatial 
  mixture-of-experts. Bioinformatics, 2025-01.

> Zhang, Z., Qiao, J., Chen, Y., & Zhou, P. (2025). Genetic Deconvolution of 
  Embryonic and Maternal Cell‐Free DNA in Spent Culture Medium of Human 
  Preimplantation Embryo Through Deep Learning. Advanced Science, e12660.


## 27/08/2025

> Han, C., Lin, S., Wang, Z., Cui, Y., Zou, Q., & Yuan, Z. (2025). Reusability 
  report: Exploring the transferability of self-supervised learning models from 
  single-cell to spatial transcriptomics. Nature Machine Intelligence, 1-15.
  https://www.nature.com/articles/s42256-025-01097-5

> Martin-Rufino, J. D., Caulier, A., Lee, S., Castano, N., King, E., Joubran, 
  S., ... & Sankaran, V. G. (2025). Transcription factor networks 
  disproportionately enrich for heritability of blood cell phenotypes. Science, 
  388(6742), 52-59. https://www.science.org/doi/10.1126/science.ads7951

> Wen, B., Wang, C., Li, K. et al. DeepMVP: deep learning models trained on 
  high-quality data accurately predict PTM sites and variant-induced 
  alterations. Nat Methods (2025). https://doi.org/10.1038/s41592-025-02797-x

> Pawlik, P., Grigoriadis, K., Bunkum, A., Coggan, H., Frankell, A. M., 
  Martinez-Ruiz, C., ... & McGranahan, N. (2025). Clone copy number diversity 
  is linked to survival in lung cancer. Nature, 1-8.
  https://www.nature.com/articles/s41586-025-09398-w

> UK Biobank Whole-Genome Sequencing Consortium. (2025). Whole-genome 
  sequencing of 490,640 UK Biobank participants. Nature.
  https://www.nature.com/articles/s41586-025-09272-9

> Xu, A. G., Xu, Y., Xing, Y., Yang, J., Bai, Y., & Tang, K. (2025). A SNP 
  Foundation Model: Application in Whole-Genome Haplotype Phasing and Genotype 
  Imputation. bioRxiv, 2025-01.

> Läderach, F., Piteros, I., Fennell, É., Bremer, E., Last, M., Schmid, S., ... 
  & Münz, C. (2025). EBV induces CNS homing of B cells attracting inflammatory 
  T cells. Nature, 1-9.
  https://www.nature.com/articles/s41586-025-09378-0

> Swanson, K., Wu, W., Bulaong, N. L., Pak, J. E., & Zou, J. (2025). The 
  Virtual Lab of AI agents designs new SARS-CoV-2 nanobodies. Nature, 1-3.
  https://www.nature.com/articles/s41586-025-09442-9

> Liao, S., Zhou, X., Liu, C., Liu, C., Hao, S., Luo, H., ... & Xu, X. (2025). 
  Stereo-cell: Spatial enhanced-resolution single-cell sequencing with 
  high-density DNA nanoball-patterned arrays. Science, 389(6762), eadr0475.
  https://www.science.org/doi/abs/10.1126/science.adr0475

> Jaganathan, K., Ersaro, N., Novakovsky, G., Wang, Y., James, T., 
  Schwartzentruber, J., ... & Farh, K. K. H. (2025). Predicting 
  expression-altering promoter mutations with deep learning. Science, eads7373.
  https://www.science.org/doi/abs/10.1126/science.ads7373



## 23/07/2025

> Bakulin, A., Levy, N., Ergen, C., Maaskola, J., & Yosef, N. 
  NOLAN: SELF-SUPERVISED FRAMEWORK FOR MAPPING CONTINUOUS TISSUE ORGANIZATION. 
  In ICLR 2025 Workshop on Machine Learning for Genomics Explorations.
  https://openreview.net/forum?id=DzlhvkuFqz

> Adduri, A., Gautam, D., Bevilacqua, B., Imran, A., Shah, R., Naghipourfar, M., ... & Roohani, Y. H. (2025). 
  Predicting cellular responses to perturbation across diverse contexts with STATE. 
  bioRxiv, 2025-06.
  https://www.biorxiv.org/content/10.1101/2025.06.26.661135v2

> Busarello, E., Biancon, G., Cimignolo, I., Lauria, F., Ibnat, Z., Ramirez, C., ... & Tebaldi, T. (2025).
  Cell Marker Accordion: interpretable single-cell and spatial omics annotation in health and disease. 
  Nature Communications, 16(1), 5399.
  https://www.nature.com/articles/s41467-025-60900-4

> Dong, M., Su, D. G., Kluger, H., Fan, R., & Kluger, Y. (2025). 
  SIMVI disentangles intrinsic and spatial-induced cellular states in spatial omics data. 
  Nature Communications, 16(1), 2990.

> Morrill Gavarró, L., Couturier, D. L., & Markowetz, F. (2025). 
  A Dirichlet-multinomial mixed model for determining differential abundance of mutational signatures. 
  BMC bioinformatics, 26(1), 59.


## 25/06/2025

> Huang, A. C., Hsieh, T. H. S., Zhu, J., Michuda, J., Teng, A., Kim, S., ... & Chu, C. (2025). 
  X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models. bioRxiv, 2025-06.

> Scherer, M., Singh, I., Braun, M. M., Szu-Tu, C., Sanchez Sanchez, P., Lindenhofer, D., ... & Velten, L. (2025). 
[Clonal tracing with somatic epimutations reveals dynamics of blood 
ageing](https://www.nature.com/articles/s41586-025-09041-8). Nature.

> Jaganathan, K., Ersaro, N., Novakovsky, G., Wang, Y., James, T., Schwartzentruber, J., ... & Farh, K. K. H. (2025). 
[Predicting expression-altering promoter mutations with deep 
learning](https://www.science.org/doi/10.1126/science.ads7373).
Science, eads7373.

> Zhang, J., Tian, R., Liu, J., Yuan, J., Zhang, S., Chi, Z., ... & Wang, D. (2024). A two-front nutrient supply environment fuels small intestinal physiology through differential regulation of nutrient absorption and host defense. Cell, 187(22), 6251-6271.

> Saraswat, M., Rueda-Gensini, L., Heinzelmann, E., Gracia, T., Memi, F., de Jong, G., ... & Mall, M. (2025). Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics. bioRxiv, 2025-05.

> Qi, J., Luo, Z., Li, C. Y., Wang, J., & Ding, W. (2025). Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks. Nature Computational Science, 1-12.

> 