# Publication Monitoring - Year 2025

## 17/09/2025

> Liu, T., Zhang, X., Ying, R., & Zhao, H. (2025). Pre-training Genomic Language 
  Model with Variants for Better Modeling Functional Genomics. bioRxiv, 2025-02.

> Gong, Q., Cao, K., Weir, J. A., Wu, Q., Li, D., & Chen, F. (2025). 
  Energy-Regularized Graph Learning for Multiscale Spatial Representation. 
  bioRxiv, 2025-08.

> SpatialZoomer: multi-scale feature analysis of spatial transcriptomics

> Zhang, N., Long, Y., Xia, S., Jia, P., Jin, Y., Chen, S., ... & Li, M. (2025). 
  A histology foundation model for high-resolution spatial omics prediction,
  tumor detection, and, clustering of spatial transcriptomics. bioRxiv, 2025-08.

> Chitra, U., Dan, S., Krienen, F., & Raphael, B. J. (2025). GASTON-Mix: a 
  unified model of spatial gradients and domains using spatial 
  mixture-of-experts. Bioinformatics, 2025-01.

> Zhang, Z., Qiao, J., Chen, Y., & Zhou, P. (2025). Genetic Deconvolution of 
  Embryonic and Maternal Cell‐Free DNA in Spent Culture Medium of Human 
  Preimplantation Embryo Through Deep Learning. Advanced Science, e12660.


## 27/08/2025

> Han, C., Lin, S., Wang, Z., Cui, Y., Zou, Q., & Yuan, Z. (2025). Reusability 
  report: Exploring the transferability of self-supervised learning models from 
  single-cell to spatial transcriptomics. Nature Machine Intelligence, 1-15.
  https://www.nature.com/articles/s42256-025-01097-5

> Martin-Rufino, J. D., Caulier, A., Lee, S., Castano, N., King, E., Joubran, 
  S., ... & Sankaran, V. G. (2025). Transcription factor networks 
  disproportionately enrich for heritability of blood cell phenotypes. Science, 
  388(6742), 52-59. https://www.science.org/doi/10.1126/science.ads7951

> Wen, B., Wang, C., Li, K. et al. DeepMVP: deep learning models trained on 
  high-quality data accurately predict PTM sites and variant-induced 
  alterations. Nat Methods (2025). https://doi.org/10.1038/s41592-025-02797-x

> Pawlik, P., Grigoriadis, K., Bunkum, A., Coggan, H., Frankell, A. M., 
  Martinez-Ruiz, C., ... & McGranahan, N. (2025). Clone copy number diversity 
  is linked to survival in lung cancer. Nature, 1-8.
  https://www.nature.com/articles/s41586-025-09398-w

> UK Biobank Whole-Genome Sequencing Consortium. (2025). Whole-genome 
  sequencing of 490,640 UK Biobank participants. Nature.
  https://www.nature.com/articles/s41586-025-09272-9

> Xu, A. G., Xu, Y., Xing, Y., Yang, J., Bai, Y., & Tang, K. (2025). A SNP 
  Foundation Model: Application in Whole-Genome Haplotype Phasing and Genotype 
  Imputation. bioRxiv, 2025-01.

> Läderach, F., Piteros, I., Fennell, É., Bremer, E., Last, M., Schmid, S., ... 
  & Münz, C. (2025). EBV induces CNS homing of B cells attracting inflammatory 
  T cells. Nature, 1-9.
  https://www.nature.com/articles/s41586-025-09378-0

> Swanson, K., Wu, W., Bulaong, N. L., Pak, J. E., & Zou, J. (2025). The 
  Virtual Lab of AI agents designs new SARS-CoV-2 nanobodies. Nature, 1-3.
  https://www.nature.com/articles/s41586-025-09442-9

> Liao, S., Zhou, X., Liu, C., Liu, C., Hao, S., Luo, H., ... & Xu, X. (2025). 
  Stereo-cell: Spatial enhanced-resolution single-cell sequencing with 
  high-density DNA nanoball-patterned arrays. Science, 389(6762), eadr0475.
  https://www.science.org/doi/abs/10.1126/science.adr0475

> Jaganathan, K., Ersaro, N., Novakovsky, G., Wang, Y., James, T., 
  Schwartzentruber, J., ... & Farh, K. K. H. (2025). Predicting 
  expression-altering promoter mutations with deep learning. Science, eads7373.
  https://www.science.org/doi/abs/10.1126/science.ads7373



## 23/07/2025

> Bakulin, A., Levy, N., Ergen, C., Maaskola, J., & Yosef, N. 
  NOLAN: SELF-SUPERVISED FRAMEWORK FOR MAPPING CONTINUOUS TISSUE ORGANIZATION. 
  In ICLR 2025 Workshop on Machine Learning for Genomics Explorations.
  https://openreview.net/forum?id=DzlhvkuFqz

> Adduri, A., Gautam, D., Bevilacqua, B., Imran, A., Shah, R., Naghipourfar, M., ... & Roohani, Y. H. (2025). 
  Predicting cellular responses to perturbation across diverse contexts with STATE. 
  bioRxiv, 2025-06.
  https://www.biorxiv.org/content/10.1101/2025.06.26.661135v2

> Busarello, E., Biancon, G., Cimignolo, I., Lauria, F., Ibnat, Z., Ramirez, C., ... & Tebaldi, T. (2025).
  Cell Marker Accordion: interpretable single-cell and spatial omics annotation in health and disease. 
  Nature Communications, 16(1), 5399.
  https://www.nature.com/articles/s41467-025-60900-4

> Dong, M., Su, D. G., Kluger, H., Fan, R., & Kluger, Y. (2025). 
  SIMVI disentangles intrinsic and spatial-induced cellular states in spatial omics data. 
  Nature Communications, 16(1), 2990.

> Morrill Gavarró, L., Couturier, D. L., & Markowetz, F. (2025). 
  A Dirichlet-multinomial mixed model for determining differential abundance of mutational signatures. 
  BMC bioinformatics, 26(1), 59.


## 25/06/2025

> Huang, A. C., Hsieh, T. H. S., Zhu, J., Michuda, J., Teng, A., Kim, S., ... & Chu, C. (2025). 
  X-Atlas/Orion: Genome-wide Perturb-seq Datasets via a Scalable Fix-Cryopreserve Platform for Training Dose-Dependent Biological Foundation Models. bioRxiv, 2025-06.

> Scherer, M., Singh, I., Braun, M. M., Szu-Tu, C., Sanchez Sanchez, P., Lindenhofer, D., ... & Velten, L. (2025). 
[Clonal tracing with somatic epimutations reveals dynamics of blood 
ageing](https://www.nature.com/articles/s41586-025-09041-8). Nature.

> Jaganathan, K., Ersaro, N., Novakovsky, G., Wang, Y., James, T., Schwartzentruber, J., ... & Farh, K. K. H. (2025). 
[Predicting expression-altering promoter mutations with deep 
learning](https://www.science.org/doi/10.1126/science.ads7373).
Science, eads7373.

> Zhang, J., Tian, R., Liu, J., Yuan, J., Zhang, S., Chi, Z., ... & Wang, D. (2024). A two-front nutrient supply environment fuels small intestinal physiology through differential regulation of nutrient absorption and host defense. Cell, 187(22), 6251-6271.

> Saraswat, M., Rueda-Gensini, L., Heinzelmann, E., Gracia, T., Memi, F., de Jong, G., ... & Mall, M. (2025). Decoding Plasticity Regulators and Transition Trajectories in Glioblastoma with Single-cell Multiomics. bioRxiv, 2025-05.

> Qi, J., Luo, Z., Li, C. Y., Wang, J., & Ding, W. (2025). Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks. Nature Computational Science, 1-12.

> 